dotCMS/src/main/java/com/dotcms/publisher/pusher/AuthCredentialPushPublishUtil.java

```diff
@@ -0,0 +1,107 @@
+package com.dotcms.publisher.pusher;
+
+import com.dotcms.auth.providers.jwt.JsonWebTokenAuthCredentialProcessor;
+import com.dotcms.auth.providers.jwt.services.JsonWebTokenAuthCredentialProcessorImpl;
+import com.dotcms.publisher.endpoint.bean.PublishingEndPoint;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.rules.parameter.display.DropdownInput;
+import com.dotmarketing.util.Config;
+import com.liferay.portal.model.User;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.IncorrectClaimException;
+import org.apache.commons.lang.StringUtils;
+import org.glassfish.jersey.server.ContainerRequest;
+import org.jetbrains.annotations.NotNull;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+
+public enum AuthCredentialPushPublishUtil {
+    INSTANCE;
+
+    private final String BEARER = "Bearer ";
```

Comment:
Codacy found an issue: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6190239)